go/kubebuilder/memcached-operator/README.md

```diff
@@ -69,6 +71,17 @@ NAME                                                               DESIRED   CUR
 replicaset.apps/memcached-operator-controller-manager-864f7c75d4   1         1         1       118s
 ```
 
```

Comment:
For we see the 3 pods we need to apply the CR ` kubectl apply -f config/samples/cache_v1alpha1_memcached.yaml -n memcached-operator-system`

Reply:
This step is missing as well.

Reply:
This step, I have made part of MAKEFILE

Reply:
solved